### PR TITLE
Create GitHub action to format using prettier

### PR DESCRIPTION
**Pull request by vzsky**
# What Changed
Add necessary files for the GitHub action

# Why
Issue #461 

Todo:
nothing ?

<details>
  <summary>📦 Published PR as canary version: <code>2.5.5-canary.538.10324.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @design-systems/babel-plugin-include-styles@2.5.5-canary.538.10324.0
  npm install @design-systems/babel-plugin-replace-styles@2.5.5-canary.538.10324.0
  npm install @design-systems/cli-utils@2.5.5-canary.538.10324.0
  npm install @design-systems/cli@2.5.5-canary.538.10324.0
  npm install @design-systems/core@2.5.5-canary.538.10324.0
  npm install @design-systems/create@2.5.5-canary.538.10324.0
  npm install @design-systems/docs@2.5.5-canary.538.10324.0
  npm install @design-systems/eslint-config@2.5.5-canary.538.10324.0
  npm install @design-systems/load-config@2.5.5-canary.538.10324.0
  npm install @design-systems/plugin@2.5.5-canary.538.10324.0
  npm install @design-systems/stylelint-config@2.5.5-canary.538.10324.0
  npm install @design-systems/utils@2.5.5-canary.538.10324.0
  npm install @design-systems/build@2.5.5-canary.538.10324.0
  npm install @design-systems/bundle@2.5.5-canary.538.10324.0
  npm install @design-systems/clean@2.5.5-canary.538.10324.0
  npm install @design-systems/create-command@2.5.5-canary.538.10324.0
  npm install @design-systems/dev@2.5.5-canary.538.10324.0
  npm install @design-systems/lint@2.5.5-canary.538.10324.0
  npm install @design-systems/playroom@2.5.5-canary.538.10324.0
  npm install @design-systems/proof@2.5.5-canary.538.10324.0
  npm install @design-systems/size@2.5.5-canary.538.10324.0
  npm install @design-systems/storybook@2.5.5-canary.538.10324.0
  npm install @design-systems/test@2.5.5-canary.538.10324.0
  npm install @design-systems/update@2.5.5-canary.538.10324.0
  # or 
  yarn add @design-systems/babel-plugin-include-styles@2.5.5-canary.538.10324.0
  yarn add @design-systems/babel-plugin-replace-styles@2.5.5-canary.538.10324.0
  yarn add @design-systems/cli-utils@2.5.5-canary.538.10324.0
  yarn add @design-systems/cli@2.5.5-canary.538.10324.0
  yarn add @design-systems/core@2.5.5-canary.538.10324.0
  yarn add @design-systems/create@2.5.5-canary.538.10324.0
  yarn add @design-systems/docs@2.5.5-canary.538.10324.0
  yarn add @design-systems/eslint-config@2.5.5-canary.538.10324.0
  yarn add @design-systems/load-config@2.5.5-canary.538.10324.0
  yarn add @design-systems/plugin@2.5.5-canary.538.10324.0
  yarn add @design-systems/stylelint-config@2.5.5-canary.538.10324.0
  yarn add @design-systems/utils@2.5.5-canary.538.10324.0
  yarn add @design-systems/build@2.5.5-canary.538.10324.0
  yarn add @design-systems/bundle@2.5.5-canary.538.10324.0
  yarn add @design-systems/clean@2.5.5-canary.538.10324.0
  yarn add @design-systems/create-command@2.5.5-canary.538.10324.0
  yarn add @design-systems/dev@2.5.5-canary.538.10324.0
  yarn add @design-systems/lint@2.5.5-canary.538.10324.0
  yarn add @design-systems/playroom@2.5.5-canary.538.10324.0
  yarn add @design-systems/proof@2.5.5-canary.538.10324.0
  yarn add @design-systems/size@2.5.5-canary.538.10324.0
  yarn add @design-systems/storybook@2.5.5-canary.538.10324.0
  yarn add @design-systems/test@2.5.5-canary.538.10324.0
  yarn add @design-systems/update@2.5.5-canary.538.10324.0
  ```
</details>

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
```
Comment (hipstersmoothie) [inline .prettierrc:2]: We would like to keep the current prettier settings. can you remove this file?

**Review by hipstersmoothie — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 1 above)
Comment (vzsky) [reply .prettierrc:2]: I didn't see the previous prettier settings in the package.json. Will remove this file then.

**Comment by vzsky** [comment]
I don't see how could I locally test this issue as it listens to PR events. 
"After you create an action, It could not run directly in your local laptop. You need to test it out in workflows. " ([yanjingzhu
](https://github.community/t/can-i-run-github-actions-on-my-laptop/17019))

Moreover, I would like to know where should I do something about _semver_ ?

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in [`v2.6.0`](https://github.com/intuit/design-systems-cli/releases/tag/v2.6.0) :rocket:

**Pull request merged**